### PR TITLE
enumerate simulation output columns

### PR DESCRIPTION
**Pull request by rmudambi**

### CODE REVIEW

File: src/vivarium_gates_bep/globals.py [hunk 1]
```diff
@@ -41,4 +44,106 @@
     'neural_tube_defects': NEONATAL_CAUSE_LIST,
 }
 
+# disease names
```
Comment (KjellSwedin) [inline src/vivarium_gates_bep/globals.py]: Consider using gbd mapping constants:
from gbd_mapping import causes
In [2]: causes.measles.name                                                                                             
Out[2]: 'measles'
In [3]: causes.meningitis.name                                                                                          
Out[3]: 'meningitis'

In [4]: causes.neonatal_preterm_birth.name                                                                              
Out[4]: 'neonatal_preterm_birth'

**Comment by rmudambi** [comment]
This is updated per the doc sent by Ali late on Friday. It may be lacking columns needed to calculate incidence and prevalence, not sure.

File: src/vivarium_gates_bep/globals.py [hunk 2]
```diff
@@ -81,45 +169,66 @@ def formatted_location(location):
...
+    f'{risk_factors.child_wasting.name}_cat1',
+    f'{risk_factors.child_wasting.name}_cat2',
+    f'{risk_factors.child_wasting.name}_cat3',
+    f'{risk_factors.child_stunting.name}_cat1',
+    f'{risk_factors.child_stunting.name}_cat2',
+    f'{risk_factors.child_stunting.name}_cat3',
     ALIVE_AT_6_MONTHS
 ]
-METRICS = [LAZ_AT_6_MONTHS, BIRTH_WEIGHT, GESTATIONAL_AGE]
+METRICS = [LAZ_AT_6_MONTHS, WHZ_AT_6_MONTHS, BIRTH_WEIGHT, GESTATIONAL_AGE]
```
Comment (KjellSwedin) [inline src/vivarium_gates_bep/globals.py]: According to Nicole " we are now referring WHZ as WLZ now for consistency (length instead of height)"

Comment (aflaxman) [reply src/vivarium_gates_bep/globals.py]: Just to be extra annoying, the public health folks might want to call it WLZ until age 1, and then change to calling it WHZ.  The customer is always right?

**Pull request closed without merging**